nntrainer/include/activation_layer.h

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0-only
+ *
+ * @file	activation_layer.h
+ * @date	17 June 2020
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug		No known bugs except for NYI items
+ * @brief	This is Activation Layer Class for Neural Network
+ *
+ */
+
+#ifndef __ACTIVATION_LAYER_H_
+#define __ACTIVATION_LAYER_H_
```

Comment:
better to use ``` __ACTIVATION_LAYER_H__```

Reply:
the last underscore ran away, replaced with new one. Thanks